docs/how-to-guides/build-and-push-feathr-registry-docker-image.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: How to build and push feathr registry docker image
+parent: Feathr How-to Guides
+---
+
+# How to build and push feathr registry docker image
+
+This doc shows how to build feathr registry docker image locally and publish to registry
+
+## Prerequisites
+
+Follow the [instructions](https://docs.docker.com/get-docker) to setup docker locally
+
+## Build docker image locally
+
+Open terminal and go to root of this repository, run following command
+
+```bash
+cd deploy
+docker build -t blrchen/feathr-sql-registry .
```

Comment:
should we use feathr/feathr-sql-registry?

Reply:
To avoid specific user name

Reply:
I remove personal username. One open issue here is feathr namespace in dockerhub is already registered by someone. @jainr is exploring available options, this docker image name might still needs to be changed later.